### DISCUSSION TITLE
Before 70,000 AD, was the moon cycle only 26 days?

### DISCUSSION THREAD

**Discussion by Fomer808** [Stellarium/stellarium#3601] [Q&A] [answered]
Hi!
      Recently, I discovered two phenomena when using software Stellarium to watch the moon.
①Starting from 60,000 BC and going forward, each month only has 28 days.  Then starting from 70,000 BC and going forward, each month only has 26 days.
 ②The moon begins to have very obvious changes in distance and distance.
     So, I'm wondering if this is a software problem or is this what real astronomical calculations are?
     Thanks !

**Comment by gzotti** [comment, marked as answer]
I have written Appendix F of the User Guide for you.

**Reply by Fomer808** [reply]
OK, thank you very much!
